src/goto-symex/symex_function_call.cpp

```diff
@@ -240,7 +240,7 @@ void goto_symext::symex_function_call_code(
   state.dirty.populate_dirty_for_function(identifier, goto_function);
 
   auto emplace_safe_pointers_result =
-    state.safe_pointers.emplace(identifier, local_safe_pointerst{ns});
+    path_storage.safe_pointers.emplace(identifier, local_safe_pointerst{});
```

Comment:
:pick: AFAICT this variable renaming should be done in a seperate commit, ideally with an explanation of why this name is more accurate (I don't know this code well enough to comment on whether this is a good renaming or not). :dove:

Reply:
Ah, no, this isn't variable renaming, it's an ownership change. `safe_pointers` used to belong to `goto_symex_statet`, but is now moved to `path_storaget`.